packages/devtools_app/lib/src/screens/performance/panes/queued_microtasks/queued_microtasks_controller.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright 2025 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+
+import 'dart:async';
+
+import 'package:devtools_app_shared/utils.dart';
+import 'package:flutter/foundation.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../../../shared/globals.dart';
+import '../../../../shared/utils/future_work_tracker.dart';
+import '../../performance_controller.dart';
+import '../../performance_model.dart';
+import '../flutter_frames/flutter_frame_model.dart';
+
+enum QueuedMicrotasksControllerStatus { empty, refreshing, ready }
+
+class QueuedMicrotasksController extends PerformanceFeatureController
+    with AutoDisposeControllerMixin {
+  QueuedMicrotasksController(super.controller) {
+    addAutoDisposeListener(_refreshWorkTracker.active, () {
+      final active = _refreshWorkTracker.active.value;
+      if (active) {
+        _status.value = QueuedMicrotasksControllerStatus.refreshing;
+      } else {
+        _status.value = QueuedMicrotasksControllerStatus.ready;
+      }
+    });
+  }
+
+  final _status = ValueNotifier<QueuedMicrotasksControllerStatus>(
+    QueuedMicrotasksControllerStatus.empty,
+  );
+  ValueListenable<QueuedMicrotasksControllerStatus> get status => _status;
+
+  final _queuedMicrotasks = ValueNotifier<QueuedMicrotasks?>(null);
+  ValueListenable<QueuedMicrotasks?> get queuedMicrotasks => _queuedMicrotasks;
+
+  final _selectedMicrotask = ValueNotifier<Microtask?>(null);
+  ValueListenable<Microtask?> get selectedMicrotask => _selectedMicrotask;
```

Comment:
order getters before private field: https://github.com/flutter/devtools/blob/master/STYLE.md#order-of-getters-and-setters